wire-schema/src/test/java/com/squareup/wire/schema/PrunerTest.kt

```diff
@@ -542,7 +542,7 @@ class PrunerTest {
   }
 
   @Test
-  fun namespacedExtensionFieldsAreRetained() {
+  fun namespaceExtensionFieldsAreRetained() {
```

Comment:
We can ignore this one it's a misunderstanding from my end.

Reply:
Let's revert this one. I don't care about the rest.

Reply:
reverted.